src/views/location.html.twig

```diff
@@ -0,0 +1,20 @@
+{% extends layout %}
+
+{% block content %}
+<h2>Location</h2>
+
+<p>We currently meet at 7:00PM on last monday of every month at <a href="https://www.google.co.uk/maps/place/Guildford+Institute/@51.2371531,-0.5724217,17z">Guildford Institute in Guildford</a>.</p>
+
+<iframe width="100%" height="450" frameborder="0" style="border:0" src="https://www.google.com/maps/embed/v1/place?q=Guildford%20Institute%2C%20Ward%20Street%2C%20Guildford%2C%20United%20Kingdom&key=AIzaSyDSWiz41MXCUkWmRxHqhxOspEwKcv9JM0Q" allowfullscreen></iframe>
```

Comment:
It seemed a fastest way at the time, want me to open a issue to fix post launch?